Mathlib/Probability/Distributions/Gaussian/Basic.lean

```diff
@@ -170,6 +170,28 @@ alias ⟨_, isGaussian_of_charFunDual_eq⟩ := isGaussian_iff_charFunDual_eq
 
 end charFunDual
 
+section charFun
+
+open InnerProductSpace
+open scoped RealInnerProductSpace
+
+variable {E : Type*} [NormedAddCommGroup E] [InnerProductSpace ℝ E] [MeasurableSpace E]
+    [CompleteSpace E] [BorelSpace E] {μ : Measure E}
+
+lemma isGaussian_iff_charFun_eq [IsFiniteMeasure μ] :
+    IsGaussian μ ↔
+    ∀ t, charFun μ t = exp (μ[fun x ↦ ⟪t, x⟫] * I - Var[fun x ↦ ⟪t, x⟫; μ] / 2) := by
+  rw [isGaussian_iff_charFunDual_eq]
+  refine ⟨fun h t ↦ ?_, fun h L ↦ by simpa using h ((toDual ℝ E).symm L)⟩
+  convert h (toDualMap ℝ E t)
+  exact charFun_eq_charFunDual_toDualMap t
+
+lemma IsGaussian.charFun_eq [IsGaussian μ] (t : E) :
+    charFun μ t = exp (μ[fun x ↦ ⟪t, x⟫] * I - Var[fun x ↦ ⟪t, x⟫; μ] / 2) := by
+  rw [isGaussian_iff_charFun_eq.1 inferInstance]
```

Comment:
This result does not need the space to be complete, right? If so, could you reorder the proofs to state it in that context?